CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ## Improvements
 - PR 1081 MNMG Renumbering - sort partitions by degree
 - PR 1115 Replace deprecated rmm::mr::get_default_resource with rmm::mr::get_current_device_resource
+- PR 1132 Upgrade CUB/Thrust to laest commits
```

Comment:
Didn't notice last time... do you want to fix the typo here 'laest' should be 'latest'

Reply:
Fixed.